### PR TITLE
Adding the arm/arm64 packages for mono runtime

### PR DESCRIPTION
**Pull request by pieandcakes**
Adding the arm/arm64 packages for the mono runtime. This is to enable
remote-ssh extension debugging.

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -1954,5 +1980,8 @@
       ],
       "binaries": []
     }
-  ]
+  ],
+  "dependencies": {
+    "vsce": "^1.74.0"
```
Comment (sean-mcmanus) [inline Extension/package.json]: Why was this added?

Comment (sean-mcmanus) [reply Extension/package.json]: It seems like it should be removed (along with the yarn.lock changes?).

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
...

**Review by WardenGnaw — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/package.json [hunk 2]
```diff
@@ -1882,6 +1882,32 @@
         "./debugAdapters/mono.linux-x86_64"
       ]
     },
+    {
+      "description": "Mono Runtime (Linux / armhf)",
+      "url": "https://aka.ms/mono-runtime-linux-arm",
```
Comment (bobbrow) [inline Extension/package.json:1698]: Should these be fwlinks instead of aka.ms links?  I don't have a strong preference, but I think it's easier to manage links when they're all on the same dashboard.

**Review by bobbrow — APPROVED** [review]
I'm ok to submit this whenever you're ready.  I just put one comment in the PR about the link type, but that's non-blocking.

File: Extension/package.json (hunk 2 above)
Comment (pieandcakes) [reply Extension/package.json:1698]: I was thinking it would be easier to be more descriptive with aka.ms links since then you know what it points to but i also didn't want to update the rest of them today.
